tasks_downstream.py

```diff
@@ -45,6 +45,7 @@
         "build"
     ]["args"]["ODOO_VERSION"]
 )
+DOCKER_COMPOSE_CMD = "docker compose"
```

Comment:
What if somebody's still using the old `docker-compose`? This would break their workflow.

Reply:
They can always set the version to one that uses docker-compose... no?

You are going to support software that is not going to be supported anymore... so there will come problems to support new versions of the docker compose manifest.

Reply:
Originally I had implemented a way to check which version is currently installed. @Tardo suggested me to remove it completely because `docker-compose` won't be supported for much long.

Reply:
I'm a bit afraid in the sense that the old `docker-compose` worked without any problems with podman.This lets me have a working machine that has no docker client or server installed.

Compose v2 is a plug-in for docker, which means you do need the docker client + plugin. Installation is more complex.

Besides, I'm currently using docker-compose v2 installed from a different package set that only provides it as a binary called `docker-compose`, avoiding the need to actually install the docker client.

Docker is banned in RHEL, Fedora and similar. Installing it there is kinda hard, while podman is available out of the box.

So, in short, if you can do a fallback, I'll be happier.

Reply:
do you have any suggestions for how to implement such change? e.g. Environment variable, or checking if the command exists every time (like this zsh plugin https://github.com/ohmyzsh/ohmyzsh/blob/master/plugins/docker-compose/docker-compose.plugin.zsh#L2 ) or other methods?

Reply:
I tried to implement a way to check which version is installed. Let's see how it works